src/components/ContributionsEpic.tsx

```diff
@@ -9,11 +9,19 @@ import { getCountryName, getLocalCurrencySymbol } from '../lib/geolocation';
 import { EpicLocalisation, EpicTracking } from './ContributionsEpicTypes';
 import { Variant } from '../lib/variants';
 
-const replacePlaceholders = (content: string, countryCode?: string): string => {
+const replacePlaceholders = (
+    content: string,
+    numArticles: number,
+    countryCode?: string,
+): string => {
     // Replace currency symbol placeholder with actual currency symbol
     // Function uses default currency symbol so countryCode is not strictly required here
     content = content.replace(/%%CURRENCY_SYMBOL%%/g, getLocalCurrencySymbol(countryCode));
 
+    // Replace number of viewed articles
+    // Value could be zero but we'll replace anyway
```

Comment:
I think we talked before about how frontend would skip rendering an epic if we don't have the value to replace a placeholder with. Is that something we want to be doing here too, perhaps with 0? I think as long as we're mirroring frontend we're good, but just wanted to raise this as a possible edge case.

Reply:
Good shout. Frontend will not replace this placeholder unless the value is truthy, but they can affort to leave unreplaced placeholders because the variants logic will check for this and stop any variant being served if it contains template variables. We have no such checks yet.
I'll going to implement that as a variant filter separately. Until then I'll merge this the existing logic of always replacing the variable, assuming it's slightly better to read `you've read 0 articles` than `you've read %%ARTICLE_COUNT%% articles.`